### ISSUE TITLE
GetToken() from auth-core returns null (with fix that works on my machine at least)

### ISSUE THREAD

**Issue by dvartdal** [nowaythatworked/auth-astro#48]
I see there is a mismatch between the session token name for auth-astro and the expected token name in auth-core.

the getToken() is expecting either: __Secure-auth.session-token or auth.session-token, if you are running on https or http. But the actually default token names with auth-astro is __Secure-auth**js**.session-token or auth**js**.session-token

Got it working by overriding the cookieName to the default naming of cookies in auth-astro when calling getToken()

```
export async function getJwtToken(req: Request) {
    const secret = import.meta.env.AUTH_SECRET
    const salt = import.meta.env.AUTH_COOKIE

    const decoded = getToken({
        req: req,
        // defaults to:
        // cookieName = secureCookie ? "__Secure-auth.session-token" : "auth.session-token",
        // Override, as naming should be __Secure-authjs.session-token for prod, authjs.session-token for dev
        cookieName: import.meta.env.AUTH_COOKIE,
        secret: secret,
        salt: salt // If you have a salt, provide it here
    })

    return decoded
}
```

Reference from auth-core (getToken())

```
export async function getToken<R extends boolean = false>(
  params: GetTokenParams<R>
): Promise<R extends true ? string : JWT | null>
export async function getToken(
  params: GetTokenParams
): Promise<string | JWT | null> {
  const {
    secureCookie,
   cookieName = secureCookie
      ? "__Secure-auth.session-token"
      : "auth.session-token",
    decode: _decode = decode,
    salt = cookieName,
    secret,
    logger = console,
    raw,
    req,
  } = params

  if (!req) throw new Error("Must pass `req` to JWT getToken()")
  if (!secret)
    throw new MissingSecret("Must pass `secret` if not set to JWT getToken()")

  const headers =
    req.headers instanceof Headers ? req.headers : new Headers(req.headers)

  const sessionStore = new SessionStore(
    { name: cookieName, options: { secure: secureCookie } },
    parse(headers.get("cookie") ?? ""),
    logger
  )

  let token = sessionStore.value

  const authorizationHeader = headers.get("authorization")

  if (!token && authorizationHeader?.split(" ")[0] === "Bearer") {
    const urlEncodedToken = authorizationHeader.split(" ")[1]
    token = decodeURIComponent(urlEncodedToken)
  }

  if (!token) return null

  if (raw) return token

  try {
    return await _decode({ token, secret, salt })
  } catch {
    return null
  }
}
```

**Comment by nowaythatworked** [comment]
Thanks a lot for the debugging! I'll implement the fix asap!

**Comment by nowaythatworked** [comment]
Hi, this issue should be fixed by [#53](https://github.com/nowaythatworked/auth-astro/pull/54). Please reopen this issue if I'm mistaken.
